roles/lib_openshift/library/oadm_manage_node.py

```diff
@@ -963,10 +964,18 @@ def _run(self, cmds, input_data):
     def openshift_cmd(self, cmd, oadm=False, output=False, output_type='json', input_data=None):
         '''Base command for oc '''
         cmds = []
-        if oadm:
-            cmds = ['oadm']
+        basecmd = 'oadm' if oadm else 'oc'
```

Comment:
Not related to this PR, but @kwoodson is there any reason that we need to keep compat with the `oadm` binary?